src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,56 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Set;
+
+public class Lotto {
+
+    private static final String NUMBER_COUNT_ERROR = "[ERROR] 6개의 숫자를 입력해주세요";
+    private static final String NUMBER_DUPLICATE_ERROR = "[ERROR] 숫자는 중복될 수 없습니다";
+    private static final String NUMBER_RANGE_ERROR = "[ERROR] 1 ~ 45 사이의 숫자를 입력해주세요";
+    private static final int MAXIMUM_NUMBER = 45;
+    private static final int MINIMUM_NUMBER = 1;
+    private static final int LOTTO_NUMBER_LIMIT = 6;
+    private final ArrayList<Integer> lotto;
+
+    public Lotto(ArrayList<Integer> numbers) {
```

Comment:
ArrayList 대신 List를 사용하는것이 좋습니다 :) 다형성을 활용하세요!
```suggestion
    private final List<Integer> lotto;

    public Lotto(List<Integer> numbers) {
```
코드 전체적으로 변경하면 좋을 거 같습니다!